src/shared/components/app/app.tsx

```diff
@@ -59,20 +64,35 @@ export class App extends Component<any, any> {
                           FirstLoadService.falsify();
                         }
 
+                        let queryProps = routeProps;
+                        if (getQueryParams) {
+                          if (this.isoData.site_res) {
+                            queryProps = {
+                              ...queryProps,
+                              ...getQueryParams(
+                                routeProps.location.search,
+                                this.isoData.site_res,
+                              ),
+                            };
+                          } else {
+                            // ErrorGuard will catch this
+                          }
+                        }
```

Comment:
If the `else` block is going to be empty, why nest the `if`s instead of just checking `getQueryParams && this.isoData.site_res`?

Reply:
This was intended to point out that a `RouteComponent` with `getQueryParams` will never see props without that don't contain the query parameters.